addons/pinniped/config-controller/README.md

```diff
@@ -19,3 +19,17 @@ To run static checks (e.g., `go fmt`, `go vet`, `golangci-lint`) and the tests:
 ```sh
 ./hack/check.sh
 ```
+
+To generate the default pinniped addon secret:
+
+```sh 
+# manually via ytt
+ytt -f ./hack/ytt/default-package-secret.yaml -f ./hack/ytt/values.yaml -v tkr=v1.23.3---vmware.1-tkg.1 -v iaas=vsphere
```

Comment:
same q as before - do we need `-f ./hack/ytt/schema` here? would either 1) removing the line so that it doesn't get out of date or 2) simply saying `ytt -f ./hack/ytt` here suffice?

Reply:
i like the shortened version 👍